Sources/UIKitNavigation/Navigation/Presentation.swift

```diff
@@ -230,14 +242,18 @@
       present: @escaping (
         _ oldValue: UIViewController?, _ newValue: UIViewController, _ transaction: UITransaction
       ) -> Void,
-      dismiss: @escaping (UIViewController, UITransaction) -> Void
+      dismiss: @escaping (
+        _ parent: UIViewController,
+        _ child: UIViewController,
+        _ transaction: UITransaction
+      ) -> Void
     ) -> ObservationToken {
       let key = UIBindingIdentifier(item)
       return observe { [weak self] transaction in
         guard let self else { return }
         if let unwrappedItem = UIBinding(item) {
           var oldController: UIViewController?
-          if let presented = presented[key] {
+          if let presented = presentedByID[key] {
```

Comment:
unrelated change, but the shadowing of `presented` made it hard to debug things.